plugin/vifm.vim

```diff
@@ -150,6 +165,45 @@ function! s:StartVifm(mods, count, editcmd, ...)
 	endif
 endfunction
 
+function! s:StartCwdJob()
+	if get(g:, 'vifm_embed_cwd', 1) && (has('job') || has('nvim'))
```

Comment:
Documentation says `False by default.`, but this seems to be `True by default.`.